staff/alex-barba/mailchimp-template/index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./style/style.css">
+    <title>Mailchimp</title>
+</head>
+<body>
+    <!-- Header Starts -->
+    <header class="header">
+        <!-- Login -->
+        <p class="header__login">Log In</p>
+        <!-- Navigation Bar -->
+        <div class="header__navigationbar">
+            <img class="header__navigationbar-logo" src="./style/main_logo.png" alt="">
+            <button class="header__navigationbar-btn">Sign Up Free</button>
+            <nav class="header__navigationbar-nav">
+                <a class="header__navigationbar-nav-pricing">Pricing</a>
```

Comment:
In tag anchors you should add href tag. If you don't have the link, you put #.

Reply:
Did not know that...Done! Thanks!